### PR TITLE
Feat/textare

### PR DESCRIPTION
**Pull request by maomaoZH**

### CODE REVIEW

File: packages/components/src/components/input/input.styles.ts [hunk 1]
```diff
@@ -102,6 +111,11 @@ export const styles: JssStyle = {
       },
     },
   },
+  'input--disable-resize': {
+    '& .input__textarea': {
+      resize: 'none',
```
Comment (eeegor) [inline packages/components/src/components/input/input.styles.ts:116]: We need to expand this to allow for multiple settings, as resize could be: `unset | none | vertical | horizontal`

Maybe we could achieve it through the component and using the `jss` `stylesheet.addRules`?

File: packages/components/src/components/input/input.tsx [hunk 2]
```diff
@@ -58,6 +63,8 @@ export class Input implements Base {
   @Prop() counter?: boolean;
   /** (optional) radio checked value */
   @Prop() preChecked?: boolean;
+  /** (optional) textarea disableResize */
+  @Prop() disableResize?: boolean;
```
Comment (eeegor) [inline packages/components/src/components/input/input.tsx:67]: Check comment above :)

**Comment by eeegor** [comment]
The vertical position of the text is not correct when used with an animated label...

image

**Comment by eeegor** [comment]
When hovering the `textarea` there is no transition on the border, please check the `input` as a reference 🙂

**Comment by maomaoZH** [comment]
The transition effect is removed when i removed transition property because when we resize the textarea, it does not look so good

**Pull request closed without merging**